apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -87,15 +87,15 @@ export class ApplicationService {
     const appModel = await this.applicationModel.create(application)
 
     //Create applicationEvent
-    const eventModel = await this.applicationEventService.create({
+    await this.applicationEventService.create({
       applicationId: appModel.id,
       state: appModel.state,
       comment: null,
     })
 
     //Create file
     if (application.files) {
-      const fileModel = await application.files.map((f) => {
+      application.files.map((f) => {
```

Comment:
removing await? Sure it works here? 😟

Reply:
Sorry was going to move it a lower level since it is basically on the wrong one here, will fix 😄